src/Angular.js

```diff
@@ -421,6 +421,20 @@ function extend(dst) {
 * Unlike {@link angular.extend extend()}, `merge()` recursively descends into object properties of source
 * objects, performing a deep copy.
 *
+* @deprecated
+* sinceVersion="1.6.5"
+* This function is deprecated, but will not be removed in the 1.x lifecycle.
+* There are a edge cases (see {@link angular.merge#known-issues known issues}) that are not
+* supported by this function. We suggest
+* using [lodash's merge()](https://lodash.com/docs/4.17.4#merge) instead.
+*
+* @knownIssue
+* This is a list of (known) object types that cannot be / are incorrectly handled by this function:
```

Comment:
The `cannot be / are incorrectly` doesn't read that well. I would split it up (e.g. `cannot be handled or are handled incorrectly`) or change it to `cannot be correctly handled` or something.